vm/actor/src/builtin/miner/policy.rs

```diff
@@ -58,7 +58,10 @@ pub const SEALED_CID_PREFIX: cid::Prefix = cid::Prefix {
 /// List of proof types which can be used when creating new miner actors
 pub fn check_supported_proof_types(proof: RegisteredSealProof) -> bool {
     use RegisteredSealProof::*;
-    matches!(proof, StackedDRG32GiBV1 | StackedDRG64GiBV1)
+    matches!(
+        proof,
+        StackedDRG2KiBV1 | StackedDRG32GiBV1 | StackedDRG64GiBV1
```

Comment:
did you just add 2kib for testing? It's not supposed to be supported

Reply:
ah, yeah good catch i forgot.